doc/content/getting-started/devmode.md

```diff
@@ -6,12 +6,57 @@ title: Development
 A Vagrant box with all the required dependencies to compile Skydive and run its
 testsuite is [available](https://app.vagrantup.com/skydive/boxes/skydive-dev).
 
-To use it :
+In devmode your host skydive source tree is synchronized with the guest skydive
+source tree, if your host $GOPATH is not set - then this will default to the
+same source tree from which you have invoked the devmode VM.
+
+The devmode guest has been successfuly tested using:
+* vagrant v2.1.1
+* VirtualBox v5.2.10
+
+OPTIONAL: If you want to explicitly specify the host skydive source
+tree, please use the following commands:
+
+```console
+export GOPATH=~/go
+echo "export GOPATH=$GOPATH" >> ~/.bashrc
+export SKYDIVE=$GOPATH/src/github.com/skydive-project
+echo "export SKYDIVE=$SKYDIVE" >> ~/.bashrc
+mkdir -p $SKYDIVE
```

Comment:
why not using golang ?
go get github.com/skydive-project/skydive

Reply:
I am fine either way (git or go get) - @lebauce / @nplanel  - please sync on this - for now I have left as-is

Reply:
@nplanel @hunchback The idea is to use the sources that are on the host. So, no need to "git" or "go get"